src/i18n.ts

```diff
@@ -49,5 +62,17 @@ const en: Record<I18N, string> = {
   [I18N.ledgerAgreementTitle]: 'Connect your Ledger',
   [I18N.ledgerAgreementText]:
     'If you have a Ledger Nano X, then you can connect it via Bluetooth to Islm Wallet. You will be able to manage funds from Ledger using Islm Wallet',
-  [I18N.ledgerAgreementAgree]: 'Connect',
+  [I18N.transactionContactEditHeaderTitle]: 'Edit Contact',
+  [I18N.settingsAccountEditHeaderTitle]: 'Edit account name',
+  [I18N.settingsContactEditHeaderTitle]: 'Contact',
+  [I18N.settingsContactEditDeleteContact]: 'Delete Contact',
+  [I18N.name]: 'Name',
+  [I18N.address]: 'Address',
+  [I18N.cancel]: 'Cancel',
+  [I18N.save]: 'Save',
+  [I18N.edit]: 'Edit',
+  [I18N.continue]: 'Continue',
```

Comment:
please, use specific screen variable names

Reply:
Why make \``I18N.${screen}Address?`\`?
Address/name/save and others are simple words that can be used on many screens